vm_server/send/test_server.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
+"""Script to create the protofile
+
+    It compiles the proto definition and
+    then creates the proto file from the text specified
+    in query1.txt or query2.txt
+"""
+import argparse
+import logging
+import os
+import sys
+import test_initialisation
+import threading
+import requests
+from flask import Flask, request
+from proto import Request_pb2
+
+
+APP = Flask(__name__)
+URL = "http://127.0.0.1:8000/assign_task"
+ROOT = ".\\"
+RESPONSE = False
+REQUEST_ID = 0
+ALL_TESTS = "all"
+TEST_1 = "1"
+TEST_2 = "2"
+TEST_3 = "3"
+TEST_4 = "4"
```

Comment:
where are these TEST_<num> variables used?

Reply:
Before using pytest testing framework. I was reading commandline flags and running tests.
They are not being used now, so I have removed them now.